astropy/utils/iers/iers.py

```diff
@@ -442,6 +442,36 @@ class because it has different requirements.
                 warn(msg, IERSDegradedAccuracyWarning)
             # No IERS data covering the time(s) and user is OK with no warning.
 
+    def _lagrange_interp(self, x_vals, y_vals, xint):
+        val = np.zeros((y_vals.shape[1:]), dtype=y_vals.dtype)
+
+        for i in range(len(x_vals)):
+            term = y_vals[i]
+            for j in range(len(x_vals)):
+                if i != j:
+                    diff = x_vals[i] - x_vals[j]
+                    diff = np.where(diff == 0, 1e-10, diff)
```

Comment:
Within the IERS table, it is guaranteed that the `x` values are never the same, so this shouldn't be necessary.